packages-backend/express/package.json

```diff
@@ -36,13 +36,13 @@
     "build": "tsc -p tsconfig.build.json"
   },
   "dependencies": {
-    "@types/node": "14.0.14",
+    "@types/node": "14.0.23",
     "express": "4.17.1",
     "express-jwt": "6.0.0",
     "jwks-rsa": "1.8.1"
   },
   "devDependencies": {
-    "@panva/jose": "1.9.3",
+    "jose": "1.27.2",
```

Comment:
I noticed by mistake that the `@panva/jose` package has been deprecated 🤷‍♂️

Reply:
Oh nice. Didn't know. We should change in the API too then :)

Reply:
yep

Reply:
It would be nice to get more prominent notifications or something about packages being deprecated. Probably best from Renovate itself.

Reply:
FYI: there seems to be some work done regarding the deprecation topic: https://github.com/renovatebot/renovate/pull/5558

Reply:
Also https://github.com/renovatebot/renovate/issues/6478